pkg/authorization/authorization.go

```diff
@@ -20,10 +20,16 @@ import (
 	dockyardsv1 "github.com/sudoswedenab/dockyards-backend/api/v1alpha3"
 	rbacv1 "k8s.io/api/rbac/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const (
+	AggregateRoleName        = "dockyards:authenticated"
+	LabelReaderAggregateRole = "rbac.dockyards.io/aggregate-to-authenticated"
```

Comment:
Reader is only a left-over right? Should probably be named `LabelAuthenticatedAggregateRole` instead.